src/main/java/ch/kalunight/zoe/repositories/CurrentGameInfoRepository.java

```diff
@@ -0,0 +1,181 @@
+package ch.kalunight.zoe.repositories;
+
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.annotation.Nullable;
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import ch.kalunight.zoe.model.dto.DTO;
+import net.rithms.riot.api.endpoints.spectator.dto.CurrentGameInfo;
+
+public class CurrentGameInfoRepository {

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The utility class name 'CurrentGameInfoRepository' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper)'](https://app.codacy.com/manual/KaluNight/Zoe-Discord-Bot/pullRequest?prid=4739924)